tsconfig.json

```
@@ -13,6 +13,7 @@
       "src": ["./src/"],
     },
     "module": "commonjs",
-    "target": "ESNext"
+    "target": "ESNext",
+    "strict": true,
```

we already have it on line 9

```suggestion
    "target": "ESNext"
```